builtin/mainmenu/pkgmgr.lua

```diff
@@ -78,23 +78,24 @@ local function load_texture_packs(txtpath, retval)
 
 	for _, item in ipairs(list) do
 		if item ~= "base" then
-			local name = item
-
 			local path = txtpath .. DIR_DELIM .. item .. DIR_DELIM
-			if path == current_texture_path then
-				name = fgettext("$1 (Enabled)", name)
-			end
-
 			local conf = Settings(path .. "texture_pack.conf")
+			local enabled = conf == current_texture_path
+
+			local name = item
```

Comment:
name = item is kinda redundant

Reply:
I thought it made it a tad clearer, but it's a minor thing so :shrug: